climada_petals/hazard/landslide.py

```diff
@@ -37,43 +37,48 @@
 LOGGER = logging.getLogger(__name__)
 HAZ_TYPE = 'LS'
 
-def sample_events_from_probs(prob_matrix, n_years, dist='binom'):
-    """sample an event set for a specified representative time span from
-    a matrix with annual occurrence probabilities
-    Draws events from chosen distribution.
+
+def sample_events(prob_matrix, n_years, dist='binom'):
+    """Repeatedly sample events"""
```

Comment:
Could you please make a proper docstring? I do not really understand why a sparse matrix is returned, and it is unclear what an event is...

Reply:
I amended the docstring. Let me know if it's understandable.